baseplate/diagnostics/tracing/__init__.py

```diff
@@ -16,12 +16,14 @@
 import requests
 from requests.exceptions import RequestException
 
-from .._compat import queue
-from ..core import (
+from ...message_queue import MessageQueue, TimedOutError
```

Comment:
:no_bell: i'm sorry for my sins. feel free to convert to absolute imports if you'd prefer.